airflow/models/dagbag.py

```diff
@@ -90,7 +90,7 @@ def __init__(
         include_examples: bool = conf.getboolean('core', 'LOAD_EXAMPLES'),
         include_smart_sensor: bool = conf.getboolean('smart_sensor', 'USE_SMART_SENSOR'),
         safe_mode: bool = conf.getboolean('core', 'DAG_DISCOVERY_SAFE_MODE'),
-        read_dags_from_db: bool = False,
+        read_dags_from_db: bool = conf.getboolean('core', 'STORE_DAG_CODE'),
```

Comment:
Hey John as mentioned on Slack, this won't be the correct fix, 

`read_dags_from_db` decides if it should read the DAGs from DB (Serialized Dags from serialized_dag table). The `conf.getboolean('core', 'STORE_DAG_CODE')` config is only used in Webserver to show the Code (as plain strings) from the `dag_code` table.